test/integration/controllermanager/seed/lifecycle/lifecycle_test.go

```diff
@@ -65,7 +66,7 @@ var _ = Describe("Seed Lifecycle controller tests", func() {
 		}
 	})
 
-	JustBeforeEach(func() {
+	BeforeEach(func() {
```

Comment:
Now we have two `BeforeEach` functions right after each other shouldn't you merge them into one?

Btw for the `when Lease object does not exist` context/test we had the following order of operations previouslly: 
```
Create Lease -> Delete Lease -> Create Seed -> Add Condition -> Check that condition is changed to unknown
```
Now the order is
```
Create Lease -> Create Seed -> Delete Lease -> Add Condition -> Check that condition is changed to oounknoown
```

There are similar changes for all tests that perform operations on the lease - previously these operations were done before the seed was created, now they are done after the fact.

Reply:
I have reverted this change. PTAL